products/sle/main.pm

```diff
@@ -7,14 +7,15 @@
 # are permitted in any medium without royalty provided the copyright
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
+
 use strict;
 use warnings;
 use testapi qw(check_var get_var get_required_var set_var check_var_array diag);
 use lockapi;
 use needle;
 use registration;
 use utils;
-use version_utils qw(is_hyperv_in_gui is_caasp is_installcheck is_rescuesystem sle_version_at_least is_desktop_installed is_jeos is_sle is_upgrade);
+use version_utils qw(is_hyperv_in_gui is_caasp is_installcheck is_rescuesystem sle_version_at_least is_desktop_installed is_jeos is_sle is_staging is_upgrade);
```

Comment:
Shouldn't `is_sles4sap()` and `is_sles4sap_standard()` be also included in this line?

Reply:
Failing test: https://openqa.suse.de/tests/1493885/file/autoinst-log.txt

Reply:
OK. Saw that `is_sles4sap` is being exported in `lib/main_common`, so I guess the fix should be to add `is_sles4sap_standard` also there.